charts/gardener-extension-provider-aws/templates/deployment.yaml

```
@@ -29,7 +29,7 @@ spec:
         {{- if and .Values.metrics.enableScraping }}
         prometheus.io/scrape: "true"
         # default metrics endpoint in controller-runtime
-        prometheus.io/port: "8080"
+        prometheus.io/port: {{ .Values.metricsPort }}
```

I think we still need to questions, otherwise it should be an invalid yaml.

```suggestion
        prometheus.io/port: "{{ .Values.metricsPort }}"
```